src/modules/GroupChannel/context/GroupChannelProvider.tsx

```diff
@@ -152,7 +150,7 @@ export const GroupChannelProvider = (props: GroupChannelProviderProps) => {
   const [fetchChannelError, setFetchChannelError] = useState<SendbirdError | null>(null);
 
   // Ref
-  const { scrollRef, scrollPubSub, scrollDistanceFromBottomRef, isScrollBottomReached, setIsScrollBottomReached } = useMessageListScroll(scrollBehavior);
+  const { scrollRef, scrollPubSub, scrollDistanceFromBottomRef, isScrollBottomReached, setIsScrollBottomReached, scrollPositionRef } = useMessageListScroll(scrollBehavior, [currentChannel]);
```

Comment:
```suggestion
  const { scrollRef, scrollPubSub, scrollDistanceFromBottomRef, isScrollBottomReached, setIsScrollBottomReached, scrollPositionRef } = useMessageListScroll(scrollBehavior, [currentChannel?.url);
```
I think having a url(simple string) would be enough for this instead of putting an huge object.

Reply:
-> https://github.com/sendbird/sendbird-uikit-react/pull/1094/commits/0d9127c226fa19e1cc6b513c2183335cca38537f